src/components/Homepage/Hero.astro

```diff
@@ -1,27 +1,108 @@
-    </p>
+<section class="relative px-4 md:px-6 md:pt-12 pt-8 md:pb-14 pb-9">
+  <div class="max-w-6xl mx-auto">
+    <div class="absolute inset-0 w-full h-full">
+      <Image
+        src={patternSvg}
+        alt="decorative pattern"
+        class="w-full h-full object-cover opacity-70"
+      />
+    </div>
+    <div class="relative text-center">
+      <h1 class="text-[2.1rem] sm:text-5xl md:text-7xl md:mb-8 mb-4 leading-tight text-white">
+        <span
+          class="bg-gradient-to-r from-[#01FD93] via-[#87FFCC] to-[#D9FFEF] inline-block text-transparent bg-clip-text"
+          >User- and developer-friendly</span
+        >
+        <br />
+        authentication based on DNS
+      </h1>
+      <p class="text-md sm:text-xl md:text-2xl text-neutral-300 mx-auto max-w-4xl">
+        VeraId is a decentralised protocol that securely attributes content to domain names.
+      </p>
+      <!-- Carousel -->
+      <div class="mt-6 md:mt-14 relative overflow-hidden w-full">
+        <div class="carousel-scroll flex gap-8 items-center" id="carousel">
+          <div class="flex gap-8 sm:gap-16 items-center min-w-max pr-4 sm:pr-8">
+            <Image src={microsoftImg} alt="Microsoft" class="h-24 w-auto object-contain" />
+            <Image src={xkcdImg} alt="XKCD" class="h-24 w-auto object-contain" />
+            <Image src={gandalfImg} alt="Gandalf" class="h-24 w-auto object-contain" />
+            <div class="relative">
+              <Image src={spotifyImg} alt="Spotify" class="h-24 w-auto object-contain" />
+              <Image
+                src={audioGif}
+                alt="Waveform gif"
+                class="absolute top-1/2 left-1/2 -translate-x-[53%] -translate-y-1/2"
+              />
+            </div>
+            <Image src={chewbaccaImg} alt="Chewbacca" class="h-24 w-auto object-contain" />
+            <Image src={bbcImg} alt="BBC" class="h-30 w-auto object-contain" />
+            <Image src={winterImg} alt="Winter" class="h-24 w-auto object-contain" />
+          </div>
+          <div class="flex gap-8 sm:gap-16 items-center min-w-max">
+            <Image src={xkcdImg} alt="XKCD" class="h-24 w-auto object-contain" />
+            <Image src={microsoftImg} alt="Microsoft" class="h-24 w-auto object-contain" />
+            <div class="relative">
+              <Image src={spotifyImg} alt="Spotify" class="h-24 w-auto object-contain" />
+              <Image
+                src={audioGif}
+                alt="Waveform gif"
+                class="absolute top-1/2 left-1/2 -translate-x-[53%] -translate-y-1/2"
+              />
+            </div>
+            <Image src={gandalfImg} alt="Gandalf" class="h-24 w-auto object-contain" />
+            <Image src={bbcImg} alt="BBC" class="h-30 w-auto object-contain" />
+            <Image src={chewbaccaImg} alt="Chewbacca" class="h-24 w-auto object-contain" />
+            <Image src={winterImg} alt="Winter" class="h-24 w-auto object-contain" />
+          </div>
```

Comment:
I think it's a very brittle hack to duplicate the carousel items. With the right code, it should just be able to loop indefinitely.